src/app/containers/PodcastPromo/index.jsx

```diff
@@ -55,11 +55,11 @@ const Promo = () => {
   ].every(Boolean);
 
   const viewTrackerRef = useViewTracker({
-    componentName: 'podcast-promo',
+    componentName: 'promo-podcast',
   });
```

Comment:
Looks good 🙂 Just a quick suggestion: I think when we added event tracking to other components we created an object like
```jsx
const eventTrackingData = {
  componentName: 'promo-podcast',
};
```
then passed this in to both `useViewTracker` and `useClickTrackerHandler` as a single source of truth, maybe worth considering here